governance/remote_executor/programs/remote-executor/src/state/posted_vaa.rs

```diff
@@ -10,7 +10,13 @@ use {
 
 // The current chain's wormhole bridge owns the VAA accounts
 impl Owner for AnchorVaa {
-    #[cfg(not(feature = "pythtest"))]
+    // do something like this
+    // #[cfg(feature = "eclipse-testnet")]
+    // fn owner() -> Pubkey {
+    //     Pubkey::from_str("HDwcJBJXjL9FpJ7UBsYBtaDjsBUhuLCUYoz3zr8SWWaQ").unwrap()
+    // }
+
+    // #[cfg(not(feature = "pythtest"))]
```

Comment:
also it's weird that we need to have both a feature flag and an env variable, is there a way to unify both?